### ISSUE TITLE
New comments?

### ISSUE THREAD

**Issue by for-coursera** [michael-lazar/rtv#527]
Is there a way to report that a thread has N new comments since the last visit? Something like `4 comments (1 new)`?

**Comment by michael-lazar** [comment]
No, there is not currently any way to do that. I'm pretty sure that's a reddit-gold only feature, right? If it's not exposed through Reddit's API, it would be difficult to implement in RTV.

**Comment by for-coursera** [comment]
> I'm pretty sure that's a reddit-gold only feature, right?

It certainly not :). Third party extensions for browsers (like `reddit-highlight-new-comments` userscript hosted [here](https://github.com/Farow/userscripts), for example) somehow achieve such functionality (it looks something like [this](https://i.imgur.com/UzNyT7y.png) in a browser).

**Comment by goggle** [comment]
Yes, this is a reddit gold feature. I guess the extensions that have this, do this by storing all the downloaded comment/post ids for a certain time span.

It would be nice if RTV could do this too.
